spec/support/sharded_model.rb

```diff
@@ -0,0 +1,24 @@
+class ShardedModel
+  include MongoMapper::Document
+
+  key :first_name, String
+  key :last_name, String
+  shard_key :first_name
```

Comment:
I've confirmed that CI fails without this configuration:
https://github.com/a2ikm/mongomapper/pull/18

Reply:
MongoDB 4.2 supports only single shard key.
So I removed `:last_name` to introduce matrix test.